hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -622,7 +622,8 @@ void setDataBlockEncoderInTest(HFileDataBlockEncoder blockEncoder) {
       for (HStoreFile storeFile : results) {
         if (compactedStoreFiles.contains(storeFile.getPath().getName())) {
           LOG.warn("Clearing the compacted storefile {} from {}", storeFile, this);
-          storeFile.getReader().close(true);
+          storeFile.getReader().close(storeFile.getCacheConf() != null ?
```

Comment:
While closing the region, doing the evict caused delay in closure. But here are we seeing some impact because of this evict? This will happen in async way from the Discharger thread right? Just trying to understand.

Reply:
Checking again.. Sorry my bad. I wanted to add this at other place but ended up here.  This is a call happening during region open. So you might want to keep ur change here.

Reply:
Oh okay. I missed that, too.. Thank you for pointing it out.